recipes/cm-unicode/meta.yaml

```diff
@@ -0,0 +1,109 @@
+    - test -f $PREFIX/fonts/cmunst.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmuntt.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunvt.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunbi.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunbso.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunbxo.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunobi.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunrb.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunso.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunsx.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmuntx.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunbl.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunbsr.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunbx.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunobx.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunrm.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunssdc.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmuntb.ttf  # [unix]
+    - test -f $PREFIX/fonts/cmunui.ttf  # [unix]
+    - if not exist $PREFIX/fonts/cmunbmo.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbtl.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunci.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunorm.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunsi.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunss.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunti.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunvi.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbbx.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbmr.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbto.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunit.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunoti.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunsl.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunst.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmuntt.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunvt.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbi.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbso.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbxo.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunobi.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunrb.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunso.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunsx.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmuntx.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbl.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbsr.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunbx.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunobx.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunrm.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunssdc.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmuntb.ttf exit 1  # [win]
+    - if not exist $PREFIX/fonts/cmunui.ttf exit 1  # [win]
```

Comment:
These windows tests aren't needed but they don't hurt.  They would need to use `\`  instead of `/`, though